src/libopensc/card-openpgp.c

```diff
@@ -2082,6 +2194,166 @@ pgp_put_data(sc_card_t *card, unsigned int tag, const u8 *buf, size_t buf_len)
 	LOG_FUNC_RETURN(card->ctx, (int)buf_len);
 }
 
+#ifdef ENABLE_OPENSSL
+/* KDF_ITERSALTED_S2K algorithm, a simplified version of openpgp_s2k from libgcrypt[1]
+ * that supports one-pass only and outputs the same key length as the hash length of
+ * the specified hash algorithm.
+ * [1]: https://github.com/gpg/libgcrypt/blob/libgcrypt-1.11.0/cipher/kdf.c#L32-L109
```

Comment:
you need to reference a specific commit if you want to get a url that is permanently accessible